synfig-core/configure.ac

```diff
@@ -347,7 +347,7 @@ AC_ARG_WITH(opengl,
 ],[
 	with_opengl="no"
 ])
-if test $with_opengl != "no" ; then {
+if test $with_opengl = "no" ; then {
```

Comment:
Looks like this will break logic?
It looks for `gl.pc`, only if you say `--with_opengl`

Reply:
Actually, no.
https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/External-Software.html

Reply:
```
./configure --without-opengl
```

```
OpenGL ---------------------------> yes


OpenCL ---------------------------> no
libdv ----------------------------> yes
OpenEXR --------------------------> yes
```

Definitely not the behavior the user expects.

Reply:
Well, I believe you :)
This commit was made on May 24 :P I just rebased and pushed now.
Maybe I changed in a bad way to autobuild include opengl support without touch it and forgot to not commit back then.
Well, I'll remove those changes. Thank you for check it (and sorry for that)

Reply:
Done